tokio-quiche/src/quic/mod.rs

```diff
@@ -0,0 +1,315 @@
+//! `async`-ified QUIC connections powered by [quiche].
+//!
+//! Hooking up a [quiche::Connection] to [tokio]'s executor and IO primitives
+//! requires an [`ApplicationOverQuic`] to control the connection. The
+//! application exposes a small number of callbacks which are executed whenever
+//! there is work to do with the connection.
+//!
+//! The primary entrypoints to set up a connection are [`listen`][listen] for
+//! servers and [`connect`] for clients.
+//! [`listen_with_capabilities`](crate::listen_with_capabilities)
+//! and [`connect_with_config`] exist for scenarios that require more in-depth
+//! configuration. Lastly, the [`raw`] submodule allows users to take full
+//! control over connection creation and its ingress path.
+//!
+//! # QUIC Connection Internals
+//!
+//! ![QUIC Worker Setup](https://github.com/cloudflare/quiche/blob/master/tokio-quiche/docs/worker.png)
```

Comment:
needs to be `https://github.com/cloudflare/quiche/blob/master/tokio-quiche/docs/worker.png?raw=true` to render as an image